### PR TITLE
Enable automated release on PyPi.

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                  password: ${{ secrets.PYPI_API_TOKEN }}
+                  repository_url: https://test.pypi.org/legacy/
+                  skip_existing: true
+
+    publish:
+
+        name: Build and publish on PyPI
+        if: startsWith(github.ref, 'refs/tags')
+
+        needs: [build]
```
Comment (mbercx) [inline .github/workflows/publish.yml:80]: If this only needs the `build` job, then why still do the `publish-test` job? Won't there be run simultaneously?

**Review by mbercx — DISMISSED** [review]
Just had a quick look (not an expert on GitHub actions), and most looks ok. Just had a small question on the use of the PyPI test.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (yakutovicha) [reply .github/workflows/publish.yml:80]: The `publish-test` will publish things on TestPyPI. It is needed to check that everything is alright, before actual publishing. At the same time the line `startsWith(github.ref, 'refs/tags/')` is probably redundant for the TestPyPI. I will remove it.

Comment (yakutovicha) [reply .github/workflows/publish.yml:77]: done.

**Comment by yakutovicha** [comment]
@mbercx please let me know if you have any additional input.

**Review by mbercx — APPROVED** [review]
Let her rip! 🚀

**Pull request merged**